### DISCUSSION TITLE
error: unrecognized arguments: --rtmp-rtmpdump=rtmpdump.exe

### DISCUSSION THREAD

**Discussion by youyouyou1234** [streamlink/streamlink#4186] [Q&A] [answered]
python-3.10.0-amd64
streamlink-2.4.0
The program works fine.
----------------------------------------------
updated to the latest version.(streamlink-3.0.1 )

error message
----------------------------------------------------------------------------------------------------------------------------------
PS C:\Users\Administrator\Desktop\3> streamlink "https://www.youtube.com/watch?v=ykvvrbqoF" best -o "abc1234.mp4"
usage: streamlink [OPTIONS] <URL> [STREAM]
streamlink: error: unrecognized arguments: --rtmp-rtmpdump=rtmpdump.exe best
PS C:\Users\Administrator\Desktop\3>
----------------------------------------------------------------------------------------------------------------

Why is this message happening?

**Comment by mkbloke** [comment, marked as answer]
https://streamlink.github.io/changelog.html#streamlink-3-0-0-2021-11-17
```
BREAKING: removed the following CLI arguments (and respective session options): (#4169, #4168)

    --rtmp-rtmpdump, --rtmpdump, --rtmp-proxy, --rtmp-timeout
    Users of Streamlink’s Windows installer will need to update their config file.
```
https://streamlink.github.io/cli.html#configuration-file

**Reply by youyouyou1234** [reply]
thanks you!
